com.unity.render-pipelines.high-definition/Runtime/Lighting/LightEvaluation.hlsl

```diff
@@ -526,20 +526,17 @@ void EvaluateLight_EnvIntersection(float3 positionWS, float3 normalWS, EnvLightD
 
 void InversePreExposeSsrLighting(inout float4 ssrLighting)
 {
-    float prevExposureInvMultiplier = GetInversePreviousExposureMultiplier();
-
-#if SHADEROPTIONS_RAYTRACING
-    if (!_UseRayTracedReflections)
-#endif
-    ssrLighting.rgb *= prevExposureInvMultiplier;
+    // Raytrace reflection use the current frame exposure - TODO: currently the buffer don't use pre-exposure.
+    // Screen space reflection reuse color buffer from previous frame
+    float exposureMultiplier = _EnableRayTracedReflections ? 1.0 : GetInversePreviousExposureMultiplier();
+    ssrLighting.rgb *= exposureMultiplier;
 }
 
 void ApplyScreenSpaceReflectionWeight(inout float4 ssrLighting)
 {
     // Note: RGB is already premultiplied by A for SSR
-#if SHADEROPTIONS_RAYTRACING
-    if (_UseRayTracedReflections)
-        ssrLighting.rgb *= ssrLighting.a;
-#endif
+    // TODO: check why it isn't consistent between SSR and RTR
+    float weight = _EnableRayTracedReflections ? 1.0 : ssrLighting.a;
+    ssrLighting.rgb *= ssrLighting.a;
```

Comment:
for SSR it is pre-multiplied in the compute shader for ray tracing it is not and there is multiples places where it could be the final input and that would be a mess to track